### PR TITLE
added a kick-start cmd to kick off rdl and removed mysql-server from image

### PR DESCRIPTION
**Pull request by cp16net**

### CODE REVIEW

**Comment by hub-cap** [comment]
+1

File: scripts/redstack [hunk 1]
```diff
@@ -378,6 +378,7 @@ function cmd_install() {
     run_devstack
     ensure_local_sources
     install_packages_2
+    exclaim "FINISHED INSTALL"
```
Comment (hub-cap) [inline scripts/redstack:381]: ALL CAPS??

Comment (cp16net) [inline scripts/redstack:381]: i thought it was important. :)

Comment (hub-cap) [inline scripts/redstack:381]: haha dork :P

**Pull request merged**